guides/k8s-microservices/src/main/paradox/index.md

```diff
@@ -0,0 +1,323 @@
+```
+
+_Refer to the files in the Chirper repository at `deploy/k8s/minikube/chirper` for more details._ 
+
+#### Deploy nginx
+
+Now that Chirper has been deployed, deploy the Ingress resouces and nginx to load the application. The command
+below will create these resources, wait for all of them to startup, and show you the cluster's pod status.
+
+```bash
+kubectl create -f deploy/k8s/minikube/nginx && \
+deploy/k8s/minikube/scripts/kubectl-wait-for-pods && \
+kubectl get pods
+```
+
+```
+ingress "chirper-ingress" created
+deployment "nginx-default-backend" created
+service "nginx-default-backend" created
+deployment "nginx-ingress-controller" created
+service "nginx-ingress" created
+NAME                                        READY     STATUS    RESTARTS   AGE
+activityservice-0                           1/1       Running   0          52s
+cassandra-0                                 1/1       Running   0          5m
+chirpservice-0                              1/1       Running   0          52s
+friendservice-0                             1/1       Running   0          52s
+nginx-default-backend-1298687872-bmhdc      1/1       Running   0          21s
+nginx-ingress-controller-1705403548-pv36b   1/1       Running   0          21s
+web-0                                       1/1       Running   0          52s
+```
+
+_Refer to the files in the Chirper repository at `deploy/k8s/minikube/nginx` for more details._ 
+
+#### Open the service in your browser
+
+Chirper and all of its dependencies are now running in the cluster. Use the following command to determine the URLs
+to open in your browser. After registering an account in the Chirper browser tab, you'll be ready to start Chirping!
+
+```bash
+echo "Chirper UI (HTTP): $(minikube service --url nginx-ingress | head -n 1)" && \
+echo "Chirper UI (HTTPS): $(minikube service --url --https nginx-ingress | tail -n 1)" && \
+echo "Kubernetes Dashboard: $(minikube dashboard --url)"
+```
+
+```
+# The URLs below will be different on your system. Be sure to
+# run the commands above to produce the correct URLs.
+
+Chirper UI (HTTP): http://192.168.99.101:31408
+Chirper UI (HTTPS): https://192.168.99.101:30122
+Kubernetes Dashboard: http://192.168.99.101:30000
+```
+
+_Note that the HTTPS URL is using a self-signed certificate so you will need to accept it to bypass any browser warnings._
+
+## Running in Production using IBM BlueMix
+
+_todo - blocked due to dependency on access to IBM's cloud service_
+
+## Conclusion
+
+Kubernetes provides many features that complement running a microservices in production. By leveraging the 
+[StatefulSet](https://kubernetes.io/docs/tutorials/stateful-application/basic-stateful-set/#objectives),
+[Ingress](https://kubernetes.io/docs/concepts/services-networking/ingress/), and 
+[Service](https://kubernetes.io/docs/concepts/services-networking/service/) resources a Lagom-based microservices 
+system can easily be deployed into your cluster.
```

Comment:
"deployed into your cluster" or "deployed as a cluster"? Or, is it the Akka cluster that is meant here, in that case, "deployed into an Akka cluster"?